app/assets/javascripts/3dbio_viewer/src/webapp/view-models/Selection.ts

```diff
@@ -99,19 +105,45 @@ function splitPdbIdb(main: string): Array<string | undefined> {
     }
 }
 
+function buildRefinedModels(items: string[]): DbItem<RefinedModelType>[] {
+    return items
+        .map(m => m.split("-"))
+        .flatMap(([id, type]) => {
+            if (
+                id &&
+                id.match(/^!{0,1}\d[\d\w]{3}$/) && //pdb regex
+                (type === "pdbRedo" || type === "cstf")
+            )
+                return [
+                    {
+                        id: `${id.replaceAll("!", "").toLowerCase()}-${type}`, //6zow-pdbRedo
+                        type,
+                        visible: id[0] !== "!",
+                    },
+                ];
+            else return [];
+        });
+}
+
 export function getSelectionFromString(items: Maybe<string>): Selection {
     const [main = "", overlay = ""] = (items || "").split(overlaySeparator, 2);
+    const overlayIds = overlay.split(mainSeparator);
+    const overlayRefined = overlayIds.filter(i => i.includes("pdbRedo") || i.includes("cstf"));
+    const overlayNotRefined = overlayIds.filter(
+        i => !(i.includes("pdbRedo") || i.includes("cstf"))
```

Comment:
No need to change it, it but this abstraction is `_.partition` -> `const [even, odd] = _.partition([1,2,3,4], x => x % 2 === 0)`